tests/yast2_gui/yast2_lan_typo.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify that yast2-lan dose not crash if there are typos in the ifcfg files.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
```

Comment:
We should use appropriate base class here, as it's test in the installed system. `y2_module_guitest` should be a better option.

Reply:
done